ethexe/consensus/src/validator/core.rs

```diff
@@ -147,14 +137,39 @@ impl ValidatorCore {
     // TODO #4741
     pub async fn aggregate_validators_commitment(
         &mut self,
-        _block: &SimpleBlockData,
+        block: &SimpleBlockData,
     ) -> Result<Option<ValidatorsCommitment>> {
-        // self.middleware.make_election_at(ElectionRequest {
-        //     at_block_hash: todo!(),
-        //     at_timestamp: todo!(),
-        //     max_validators: todo!(),
-        // });
-        Ok(None)
+        let SimpleBlockData { hash, header } = block;
+        let timelines = self
```

Comment:
I think better make timelines to be field of ValidatorCore and take this once from db at the service construction.